upgrade_tests/test_existance_relations/cli/test_contenthosts.py

```diff
@@ -32,7 +32,7 @@
 
 
 # Tests
-@dont_run_to_upgrade(['6.3', '6.4', '6.5', '6.6', '6.7'])
+@dont_run_to_upgrade(['6.3', '6.4', '6.5', '6.6', '6.7', '6.8'])
```

Comment:
I think we shld now be able to remove old / End of support sat versions from this list, as these tests will never be re-running on those old sat versions.

Reply:
yes, you are right, but I think it will help to track the all unsupported version of the respective test component.